packages/popup/src/pages/CreateSeed/NewSeed.js

```diff
@@ -1,5 +1,28 @@
+import {useTranslation} from 'react-i18next'
+import create from 'zustand'
+import Input from '@cfxjs/component-input'
+import {CompWithLabel} from '../../components'
+
+const usePageStore = create(set => ({
+  groupName: '',
+  setGroupName: groupName => set({groupName}),
+}))
+
 function NewSeed() {
-  return <div>NewSeed</div>
+  const {t} = useTranslation()
```

Comment:
why not use the improved zustand for this

Reply:
done